jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/ExtendedMemoryMXBeanImpl.java

```diff
@@ -84,32 +86,33 @@ protected void startNotificationThread() {
 				return thread;
 			};
 
-			Thread notifier = AccessController.doPrivileged(createThread);
+			Thread notifier = java.security.AccessController.doPrivileged(createThread);
 			notifier.start();
 		}
 	}
 
 	/**
 	 * Deprecated. Use getTotalPhysicalMemorySize().
 	 */
-	/*[IF Sidecar19-SE]
+	/*[IF JAVA_SPEC_VERSION > 8]*/
 	@Deprecated(forRemoval = true, since = "1.8")
-	/*[ELSE]*/
+	@SuppressWarnings("deprecation")
+	/*[ELSE] JAVA_SPEC_VERSION > 8 */
 	@Deprecated
-	/*[ENDIF]*/
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemory() {
-		return osinstance.getTotalPhysicalMemorySize();
+		return getTotalPhysicalMemorySize();
 	}
 
+	/*[IF JAVA_SPEC_VERSION > 8]*/
+	@SuppressWarnings("deprecation")
+	/*[ENDIF] JAVA_SPEC_VERSION > 8 */
 	public long getTotalPhysicalMemorySize() {
```

Comment:
Just like getUsedPhysicalMemory() is removed (but I suggest deprecated), and getTotalPhysicalMemory() is deprecated, getTotalPhysicalMemorySize() should be deprecated as well.

Reply:
Fixed in b2749a307fcf0b8f54344aa21e73327e1edb4a74.